eng/pipelines/spec-gen-sdk.yml

```diff
@@ -32,9 +32,9 @@ parameters:
     
 trigger: none
 pr:
-  branches:
+  paths: 
     include:
-      - user/spec-gen-pipeline-testing
+      - specification/**
```

Comment:
Not an issue right now but when we decide to make these checks required we will need to trigger on all files because otherwise this required check will not trigger and complete.